### ISSUE TITLE
Cluster Generalization

### ISSUE THREAD

**Issue by hyperturing** [zonca/nbsubmit#1]
Is there a way that cluster selection could be generalized beyond Comet? We are using the Bridges system at PSC, and are looking at this tool as a potential way to remotely submit our notebooks to Bridges.

**Comment by zonca** [comment]
yes, Bridges also uses Slurm, you can add another `if cluster_name==` at https://github.com/zonca/nbsubmit/blob/master/nbsubmit/cluster.py#L241, please test it out, open a pull request with your initial attempt and report any error you encounter

**Comment by zonca** [comment]
another option is #2 , however I don't have any free time to pursue that for the next few months, but you might want to consider using `parsl` as well
